examples/pump-app/cc13x2x7_26x2x7/main/AppTask.cpp

```diff
@@ -357,17 +363,33 @@ void AppTask::DispatchEvent(AppEvent * aEvent)
     }
 }
 
+void AppTask::InitOnOffClusterState()
+{
+
+    EmberStatus status;
+
+    ChipLogProgress(NotSpecified, "Init On/Off clusterstate");
+
+    // Write false as pump always boots in stopped mode
+    status = OnOff::Attributes::OnOff::Set(ONOFF_CLUSTER_ENDPOINT, false);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+
```

Comment:
This blank line was one of the ones @andy31415 was asking about....

Reply:
Not going my way today. Should be fixed now.